README.md

```diff
@@ -29,13 +29,88 @@ on. You will need:
 See our [tracking issue](https://github.com/bitcoin-dev-project/sim-ln/issues/26)
 for updates on implementation support (contributions welcome!).
 
-## Configuration
-The simulator is configured with two sets of information expressed in 
-json: 
-* `nodes`: a set of nodes that you have permission to execute 
-  commands on.
-* `activity`: a description of the payment activity that you would 
-  like to generate.
+## Getting Started
+Clone the repo: 
+```
+git clone https://github.com/bitcoin-dev-project/sim-ln
+cd sim-ln
+```
+
+Install the CLI: 
+```
+cargo install --locked --path sim-cli/
+```
+
+Run Simulation with Config (see [configuration](#configuration) for details):
+ ```
+sim-cli --config config.json
+```
+
+### Configuration
+The simulator requires access details for a set of `nodes` that you 
+have permission to execute commands on. Note that the current version 
+of the simulator uses keysend to execute payments, which must be 
+enabled in LND using `--accept-keysend`.
+
+Payment activity can be simulated in two different ways:
+* Random activity: generate random activity on the `nodes` provided, 
+  using the graph topology to determine payment frequency and size.
+* Configured activity: provide descriptions of specific payments that 
+  you would like the generator to execute.
+
+### Configuration - Random Activity
+
+To run the simulator with random activity generation, you just need to 
+provide a set of nodes and the simulator will generate activity based 
+on the topology of the underlying graph. Note that payments will only 
+be sent between the `nodes` that are configured so that liquidity does 
+not "drain" from the simulation.
+
+```
+{
+  "nodes": [
+    {
+      "LND": {
+        "id": "Alice",
+        "address": "https://localhost:10011",
+        "macaroon": "/path/admin.macaroon",
+        "cert": "/path/tls.cert"
+      }
+    },
+    {
+      "CLN": {
+        "id": "0230a16a05c5ca120136b3a770a2adfdad88a68d526e63448a9eef88bddd6a30d8",
+        "address": "https://localhost:10013",
+        "ca_cert": "/path/ca.pem",
+        "client_cert": "/path/client.pem",
+        "client_key": "/path/client-key.pem"
+      }
+    }
+  ]
+}
+```
+
+Nodes can be identified by an arbitrary string ("Alice", "CLN1" etc) or
+by their node public key. If a valid public key is provided it *must* 
+match the public key reported by the node.
+
+There are a few cli flags that can be used to toggle the characteristics 
+of the random activity that is generated: 
+* `--expected-payment-amount`: the approximate average amount that 
+  will be sent by nodes, randomness will be introduced such that larger
+  nodes send a wider variety of payment sizes around this expectation.
+* `--capacity-multiplier`: the number of times over that each node in 
+  the network sends their capacity in a calendar month - 2 indicating 
```

Comment:
Using `-` style here reads pretty weird

Reply:
Meaning, don't use ` to make it a code block?

Reply:
Oh no, I mean "-2 indicating ..."

Looks like you're talking to minus two

Reply:
ah gotcha!